browser/src/canvas/sections/CommentListSection.ts

```diff
@@ -2359,9 +2359,8 @@ export class CommentSection extends app.definitions.canvasSectionObject {
 			this.checkSize();
 			this.update();
 		}
-
-		var show = this.map.stateChangeHandler.getItemValue('showannotations');
-		this.setView(show === true || show === 'true');
+		var initalShowAnnotations = commentList.length > 0;
+		app.map.showComments(initalShowAnnotations);
```

Comment:
This makes it out of sync when inserting a comment to document which didn't have comments before.
Then you have to click 2 times "show comments" to acctually toggle.

I think it should stay as it was (default is on) - less problems.

As improvement we can probably remember the last state which user had in the local browser settings?  cc: @pedropintosilva 
Or maybe it is saved in the document? Could you try with odt and docx, if they remember the last status after we save and reopen document in LibreOffice?